package.json

```diff
@@ -26,16 +27,19 @@
     "@types/vscode": "~1.67.0",
     "@typescript-eslint/eslint-plugin": "~6.4.1",
     "@typescript-eslint/parser": "~6.4.1",
-    "@vitest/coverage-v8": "~0.34.2",
-    "@vitest/ui": "~0.34.2",
+    "@vitest/coverage-v8": "~1.0.0",
+    "@vitest/ui": "~1.0.0",
     "concurrently": "~8.2.1",
     "esbuild": "~0.19.2",
     "eslint": "~8.47.0",
     "eslint-plugin-header": "~3.1.1",
     "editorconfig": "~2.0.0",
     "shx": "~0.3.4",
     "typescript": "~5.1.6",
-    "vitest": "~0.34.2"
+    "vitest": "~1.0.0"
+  },
+  "overrides": {
+   "@types/node": "~16.18.41"
```

Comment:
Question: What is this override for? We already use `@types/node@16.18.41` in our `devDependencies`.

Reply:
I stated it in the commit msg for the records:
> added version override on '@types/node' to cope with vitest's (transitive) peer dependency on '@types/node@"^18.0.0 || >=20.0.0"'

I think vitest's dependency is even flagged as optional, appears to work nicely.

Reply:
I don't really understand why they have a `@types/*` dependency as a peer dependency (they really shouldn't), but I understand now why you did what you did. Thanks.